src/hooks/useSettings.ts

```diff
@@ -0,0 +1,50 @@
+import { atom, useAtom } from "jotai";
+import { useEffect } from "react";
+
+export type Settings = {
+  title: string;
+  key: string;
+  type: string;
+  group: string;
+  value: any;
+  setValue: any;
+};
+
+const nodeDelayDefault = atom(0);
+const maximumRuntimeDefault = atom(3000);
+const settingsListDefault = atom([{} as Settings]);
```

Comment:
Instead this could be written as follows: `const settingsListDefault = atom<Settings[]>([]);